### ISSUE TITLE
gulp extend published npm themelet not working

### ISSUE THREAD

**Issue by paulocfjunior** [liferay/liferay-theme-tasks#87]
How to reproduce:
- Create a theme with `yo liferay-theme`
- Go to the theme folder
- Run `gulp extend`
    - Where would you like to search for themelets?
      Choose `Search npm registry (published modules)`
    - Search npm for themelets
      Type `efx-buttons-themelet` (with hiphens)

Expected:
    The task **installs** or **uninstalls** the themelet into/from the theme.

Current behavior:
    The operation could not finish and return the 404 error, as below.
```
	(node:6099) UnhandledPromiseRejectionWarning: HTTPError: Response code 404 (Not Found)
	    at /home/paulo/Documentos/themelet/my-liferay-theme/node_modules/got/index.js:112:13
	    at BufferStream.<anonymous> (/home/paulo/Documentos/themelet/my-liferay-theme/node_modules/read-all-stream/index.js:64:3)
	    at emitNone (events.js:111:20)
	    at BufferStream.emit (events.js:208:7)
	    at finishMaybe (/home/paulo/Documentos/themelet/my-liferay-theme/node_modules/read-all-stream/node_modules/readable-stream/lib/_stream_writable.js:630:14)
	    at endWritable (/home/paulo/Documentos/themelet/my-liferay-theme/node_modules/read-all-stream/node_modules/readable-stream/lib/_stream_writable.js:638:3)
	    at BufferStream.Writable.end (/home/paulo/Documentos/themelet/my-liferay-theme/node_modules/read-all-stream/node_modules/readable-stream/lib/_stream_writable.js:594:41)
	    at IncomingMessage.onend (_stream_readable.js:595:10)
	    at Object.onceWrapper (events.js:313:30)
	    at emitNone (events.js:111:20)
	    at IncomingMessage.emit (events.js:208:7)
	    at endReadableNT (_stream_readable.js:1064:12)
	    at _combinedTickCallback (internal/process/next_tick.js:138:11)
	    at process._tickCallback (internal/process/next_tick.js:180:9)

	(node:6099) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
	(node:6099) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

The themelet is published to [npm](https://www.npmjs.com/package/efx-buttons-themelet) and working properly (tested locally).

**Comment by paulocfjunior** [comment]
Submitted possible solution on #88.

**Comment by jbalsas** [comment]
Hey @paulocfjunior, we're moving this project to https://github.com/liferay/liferay-themes-sdk.

This issue was also reported as https://github.com/liferay/liferay-themes-sdk/issues/17, fixed an shipped in the new `8.0.0-beta.1` version!
